manifests/repo.pp

```diff
@@ -155,7 +164,6 @@
 
   # Only need to deal with importing GPG keys, if we have gpgcheck enabled
```

Comment:
Need a conditional around the existing conditional and code for yum::rpm_gpg_key

```puppet
if $ensure == 'present' {
<existing code>
}
```

Reply:
Added ensure == 'present' condition.